src/eugenie_solutions/week2/DivisibleBy3_5_15.java

```diff
@@ -0,0 +1,64 @@
+package eugenie_solutions.week2;
+
+import java.util.Scanner;
+
+public class DivisibleBy3_5_15 {
+    public static void main(String[] args) {
+
+        String divisibelBy15 = "";
+        String divisibelBy5 = "";
+        String divisibelBy3 = "";
+
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter number between 1 ~ N");
+        int number = scan.nextInt();
+        while (number <= 0) {
+            System.out.println("Invalid number. Please re-enter");
+            number = scan.nextInt();
+        }
+
+        for (int i = 1; i <= number; i++) {
+            if (i % 3 == 0 && i % 5 == 0 && i % 15 == 0) {
+                divisibelBy15 += " " + i;
+            }
+            if (i % 3 == 0 && i % 15 != 0) {
+                divisibelBy3 += " " + i;
+            }
+            if (i % 5 == 0 && i % 15 != 0) {
+                divisibelBy5 += " " + i;
```

Comment:
Looks good but, i would say that the logic for finding numbers divisible by 15, 5, and 3 could be simplified. Instead of separate if statements, you could use a single if-else structure to check for divisibility by 15 first, then by 5, and finally by 3